InnerEye/ML/config.py

```diff
@@ -480,6 +480,12 @@ class SegmentationModelBase(ModelConfigBase):
                                                  "of the first N subjects in the training set will be "
                                                  "written to the outputs folder.")
 
+    #: Some loss functions (e.g. SoftDice) may produce results on overlapping labels, but others (e.g. FocalLoss) will
+    #: fail with a cryptic error message. If you are sure that you want to use mutually exclusive labels then set this
```

Comment:
I think you meant "if you are sure you want to use labels that are NOT mutually exclusive"?

Reply:
I did! Fixed.